src/components/LandingPage/TwitterWidget.tsx

```diff
@@ -7,73 +7,73 @@ import Button from '../General/Button'
 const TwitterWidget: React.FC = () => {
   const { t } = useTranslation()
 
-  const arr = [
+  const tweets = [
     '1393942312812691461',
     '1394964427074646020',
     '1386625779577106433',
     '1393679448600363016',
     '1393623123443036165',
   ]
-  const [firstTweetId, setFirstTweetId] = useState(0)
-  const [secondTweetId, setSecondTweetId] = useState(1)
+  const [carouselPosition, setCarouselPosition] = useState(0)
 
-  const totalTweets = arr.length
+  const tweetsPerSlide = 1
+  const totalSlides = Math.ceil(tweets.length / tweetsPerSlide)
 
-  const nextTweet = () => {
-    setFirstTweetId(secondTweetId)
-    if (secondTweetId >= totalTweets - 1) {
-      setSecondTweetId(0)
-    } else {
-      setSecondTweetId(secondTweetId + 1)
-    }
+  const nextTweets = () => {
+    if (carouselPosition === totalSlides - 1) return
+    setCarouselPosition(carouselPosition + 1)
   }
 
-  const previousTweet = () => {
-    setSecondTweetId(firstTweetId)
-    if (firstTweetId === 0) {
-      setFirstTweetId(totalTweets - 1)
-    } else {
-      setFirstTweetId(firstTweetId - 1)
-    }
+  const previousTweets = () => {
+    if (carouselPosition === 0) return
+    setCarouselPosition(carouselPosition - 1)
   }
 
+  const getTweetsToShow = () => {
+    const tweetCards = []
```

Comment:
@alaa-yahia you had to use `.map` cause you can't use `for` loop inside JSX (same we can't use if/else and have to use terniary) cause in JSX (read difference between statements and expressions in JS to see why)